README.md

```diff
@@ -46,25 +46,27 @@ For deb-based platforms (e.g. Ubuntu and Debian) run the following to add the
 repo key and setup a new sources.list entry:
 
 ```shell
-# influxdb.key GPG Fingerprint: 05CE15085FC09D18E99EFB22684A14CF2582E0C5
-wget -q https://repos.influxdata.com/influxdb.key
-echo '23a1c8836f0afc5ed24e0486339d7cc8f6790b83886c4c96995b88a061c5bb5d influxdb.key' | sha256sum -c && cat influxdb.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/influxdb.gpg > /dev/null
-echo 'deb [signed-by=/etc/apt/trusted.gpg.d/influxdb.gpg] https://repos.influxdata.com/debian stable main' | sudo tee /etc/apt/sources.list.d/influxdata.list
+# influxdata-oss-archive_compat.key GPG fingerprint:
+#     9D53 9D90 D332 8DC7 D6C8 D3B9 D8FF 8E1F 7DF8 B07E
+wget -q https://repos.influxdata.com/influxdata-oss-archive_compat.key
+echo '393e8779c89ac8d958f81f942f9ad7fb82a25e133faddaf92e15b16e6ac9ce4c influxdata-oss-archive_compat.key' | sha256sum -c && cat influxdb.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/influxdata-oss-archive_compat.key > /dev/null
+echo 'deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-oss-archive_compat.key] https://repos.influxdata.com/debian stable main' | sudo tee /etc/apt/sources.list.d/influxdata.list
```

Comment:
`/etc/apt/trusted.gpg.d/influxdata-oss-archive_compat.key` should be `/etc/apt/trusted.gpg.d/influxdata-oss-archive_compat.gpg`